docs/quarantine.md

```diff
@@ -55,9 +55,10 @@ A test must be put in quarantine when any of these conditions is met:
 
 A PR will be proposed on Mondays every two weeks with a batch of the tests that
 met the first condition. A PR can be proposed at any time for the tests that meet
-the second condition. In both cases the PR will add the text `[QUARANTINE]` to
-each test's description in the code. An email will be sent to the owners of the
-suspected tests.
+the second condition. In both cases the PR will add the text `[QUARANTINE]` and
+the `decorators.Quarantine` [labelDecorator](https://github.com/kubevirt/kubevirt/blob/9a3799f7a0b97b70033e119c0b401778c51dee14/tests/decorators/decorators.go#L5)
+to each test's description in the code.
+An email will be sent to the owners of the suspected tests.
```

Comment:
Good point to notify the owners, but that brings up another question: how do we find the owner of a test. `git annotate` might not be enough, since the authors might not be active any more? Ideas?

FYI @aburdenthehand @fabiand

Reply:
This part was already there! I think that the owner here is referring to the proper `sig`

Reply:
I see. Thanks!